python/cugraph/structure/graph.py

```diff
@@ -565,945 +277,394 @@ def from_numpy_matrix(self, np_matrix):
-        vertex_subset.
-
-        Parameters
-        ----------
-        vertex_subset : cudf.Series or iterable container, optional
-            A container of vertices for displaying corresponding in-degree.
-            If not set, degrees are computed for the entire set of vertices.
-
-        Returns
-        -------
-        df : cudf.DataFrame
-            GPU DataFrame of size N (the default) or the size of the given
-            vertices (vertex_subset) containing the in_degree. The ordering is
-            relative to the adjacency list, or that given by the specified
-            vertex_subset.
-
-            df[vertex] : cudf.Series
-                The vertex IDs (will be identical to vertex_subset if
-                specified).
-            df[degree] : cudf.Series
-                The computed in-degree of the corresponding vertex.
-
-        Examples
-        --------
-        >>> M = cudf.read_csv('datasets/karate.csv', delimiter=' ',
-        >>>                   dtype=['int32', 'int32', 'float32'], header=None)
-        >>> G = cugraph.Graph()
-        >>> G.from_cudf_edgelist(M, '0', '1')
-        >>> df = G.in_degree([0,9,12])
+        return self.graph_properties.directed
 
+    def is_renumbered(self):
         """
-        return self._degree(vertex_subset, x=1)
-
-    def out_degree(self, vertex_subset=None):
+        Returns True if the graph is renumbered.
         """
-        Compute vertex out-degree. Vertex out-degree is the number of edges
-        pointing out from the vertex. By default, this method computes vertex
-        degrees for the entire set of vertices. If vertex_subset is provided,
-        this method optionally filters out all but those listed in
-        vertex_subset.
-
-        Parameters
-        ----------
-        vertex_subset : cudf.Series or iterable container, optional
-            A container of vertices for displaying corresponding out-degree.
-            If not set, degrees are computed for the entire set of vertices.
-
-        Returns
-        -------
-        df : cudf.DataFrame
-            GPU DataFrame of size N (the default) or the size of the given
-            vertices (vertex_subset) containing the out_degree. The ordering is
-            relative to the adjacency list, or that given by the specified
-            vertex_subset.
-
-            df[vertex] : cudf.Series
-                The vertex IDs (will be identical to vertex_subset if
-                specified).
-            df[degree] : cudf.Series
-                The computed out-degree of the corresponding vertex.
-
-        Examples
-        --------
-        >>> M = cudf.read_csv('datasets/karate.csv', delimiter=' ',
-        >>>                   dtype=['int32', 'int32', 'float32'], header=None)
-        >>> G = cugraph.Graph()
-        >>> G.from_cudf_edgelist(M, '0', '1')
-        >>> df = G.out_degree([0,9,12])
+        return self.properties.renumbered
```

Comment:
self.properties is not defined in `Graph`. Maybe `self.Impl.properties.renumbered` since `renumbered` exists in `simpleGraph` assuming `self.Impl` is `not None` ?